main.tex

```diff
@@ -230,37 +235,37 @@
 
 \item  not interfere with any closed-circuit television monitoring devices, audio-visual cameras and other audio-visual surveillance devices; and
 
-\item  not to do or permit anything to be done that may prejudice the security or safety of the Common Property or any person in or around the Building.
+\item  not do (or permit to be done) anything that may harm the security or safety of the Common Property or any fellow Lot Owner or Occupier or their pets in or around the Building.
 
 \end{enumerate}
 
-\item  Without limiting the generality of the requirement of Rule 1.3(2)(a), an Owner or Occupier must not:
+\item  Without limiting the generality of the requirement of Rule 1.3(3)(a), an Owner or Occupier must not:
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  allow any unknown guest(s) to enter the Building;
-
-\item  allow anyone access to the foyer or car park if they have not used normal security procedures to enter;
+\item  allow anyone access to the foyer or garage if they have not used normal security procedures to enter;
 
 \item  allow their guest/invitees to access the Building or use the common area facilities unless they are accompanied by an Owner or Occupier of a Lot.
 
 \end{enumerate}
 
-\item  All Security Keys remain the property of the individual Owner and an Occupier must:
+\item  All Security Keys remain the property of the individual Owner.
+
+\item In regards to Security Keys, an Owner or Occupier must:
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  keep the Security Keys and/or remote control which has been allocated, safe;
+\item  keep the allocated Security Keys and/or remote control safe;
 
 \item  immediately report the loss of any Security Key to the Owners Corporation;
 
-\item  immediately noting the Owners Corporation of any person who is not an Owner or Occupier of a Lot to whom a Security Key has been made available;
+\item  immediately notify the Owners Corporation of a Security Key being available to any person who is not an Owner or Occupier of a Lot;
 
 \item  not copy any Security Key;
 
-\item  ensure that only the Owners Corporation replace any Security Key;
```

Comment:
The Owners Corporation is not abiding by this rule. They are not replacing Security Keys.